index.js

```diff
@@ -99,6 +100,8 @@ alfy.config = new Conf({
 	cwd: alfy.alfred.data
 });
 
+alfy.userConfig = new AlfredConfig();
```

Comment:
Ok, so apparently I have to fix something upstream first. Because I do `path.dirname(getEnv())` instead of `getEnv()` 🤦 .

https://github.com/SamVerschueren/alfred-config/blob/master/index.js#L11

Reply:
Done